src/modelbench/benchmark_runner.py

```diff
@@ -336,7 +353,9 @@ def handle_item(self, item: TestRunItem) -> TestRunItem:
         return item
 
     def collect_annotations(self, item):
-        for annotator_key, annotator in item.test.get_annotators().items():
+        # TODO: Parallelize annotators
+        test_annotators = self.test_run.test_annotators[item.test.uid]
```

Comment:
This might be better as a convenience method in test_run, like `annotators_for(test)` or `annotators_for(test_item)`.

Reply:
Okay! Updated.